core/pta/ocall.c

```diff
@@ -0,0 +1,364 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2020, Microsoft Corporation
+ */
+
+/*
+ * TA Interface:
+ * -------------
+ * params[0]: VALUE IN/OUT
+ *   IN: < HOST CMD ID,  Param Types   >
+ *  OUT: < CMD Ret Val, CMD Ret Origin >
+ * params[1]: < [INOUT] TEE_Param[] >
```

Comment:
Thank you for your suggestions. I've rearranged the comment following your idea.

I also removed the OCALL param types from the TA interface; they aren't needed there since they are already included in `struct utee_params`.